database/schema.sql

```diff
@@ -207,6 +207,72 @@ CREATE TABLE rotation_move_history (
   FOREIGN KEY (new_bucket) REFERENCES rotation_buckets (bucket_id)
 );
 
+-- BUDGET TABLES
+
+-- Table of all accounts Ruddock owns.
+CREATE TABLE budget_accounts (
+  account_id INTEGER NOT NULL AUTO_INCREMENT,
+  account_name VARCHAR(32) NOT NULL,
+  initial_balance NUMERIC(9,2) NOT NULL, -- this is sufficent to store up to 10 million
```

Comment:
Why not current_balance? You could have "initial balance" just be the first transaction recorded.